apps/src/code-studio/progressRedux.js

```diff
@@ -28,29 +33,27 @@ const SET_CURRENT_STAGE_ID = 'progress/SET_CURRENT_STAGE_ID';
 const SET_SCRIPT_COMPLETED = 'progress/SET_SCRIPT_COMPLETED';
 const SET_STAGE_EXTRAS_ENABLED = 'progress/SET_STAGE_EXTRAS_ENABLED';
 const USE_DB_PROGRESS = 'progress/USE_DB_PROGRESS';
-const OVERWRITE_PROGRESS = 'progress/OVERWRITE_PROGRESS';
-
-const PEER_REVIEW_ID = -1;
+const OVERWRITE_PROGRESS_RESULTS = 'progress/OVERWRITE_PROGRESS_RESULTS';
 
 const initialState = {
   // These first fields never change after initialization
   currentLevelId: null,
   currentStageId: null,
+  currentLesson: null,
   professionalLearningCourse: null,
   // used on multi-page assessments
   saveAnswersBeforeNavigation: null,
-  stages: null,
   lessonGroups: null,
```

Comment:
Should the default value of lessonGroups now be an empty array instead of null? I'm running into lots of issues in the tests because the code now assumes lessonGroups is an array and isn't handling null gracefully.

Reply:
yeah that makes sense!